confirmed_malicious.rego

```diff
@@ -1,27 +1,32 @@
-package policy
+# METADATA
+# title: Confirmed Malicious
+# description: |
+#   Return a violation if the pacakge or author is tied to known malicious behavior
+
```

Comment:
I think the linter is also complaining about the new line here. 
```suggestion
```

Reply:
I fixed this issue